Banyan/src/sessions.jl

```diff
@@ -112,6 +114,7 @@ function start_session(;
     end
     s3_bucket_name = get_cluster_s3_bucket_name(cluster_name)
 
+    # TODO: Construct this dictionary
```

Comment:
I think you can take this comment off, since this is no longer a TODO!

Reply:
K